Rb.Forms.Barcode.Pcl/BarcodeScanner.cs

```diff
@@ -246,6 +246,10 @@ public void OnCameraReleased()
 
         private static void OnBarcodeChanged(BindableObject bindable, Barcode oldValue, Barcode newBarcode)
         {
+            if (newBarcode == null) {
+                return;
+            }
```

Comment:
Fokke and I talked about this, and it seems that the use-case I described above is still handled (notification of no more barcode in frame).  :+1: